lang/node-serialport/Makefile

```diff
@@ -56,6 +56,7 @@ define Build/Compile
 	npm install --build-from-source --target_arch=$(CPU) -g \
 		--prefix="$(PKG_INSTALL_DIR)/usr/" \
 		`npm pack $(PKG_BUILD_DIR) | tail -n 1`
+	if test -d ~/npm-global; then ls -alR ~/npm-global/; false; else true; fi
```

Comment:
I think we all would prefer it if one does not put "debug" code in their submissions for us to review - and waste time.

You should be able to test in your own environment by either building with the SDK or setting up your own CircleCI account and use a test branch if you want to debug with our CI scripts and setup.

Reply:
Sorry about that.  I've never used the CircleCI before, and don't know how to mimic the buildbots exactly, and that's why I sometimes struggle to fix some build failures.  Can you give me a pointer to get started?